mail-client/mutt-wizard/mutt-wizard-3.3.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit optfeature
+
+DESCRIPTION="A system for automatically configuring neomutt and isync"
+HOMEPAGE="https://github.com/LukeSmithxyz/mutt-wizard"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/LukeSmithxyz/mutt-wizard.git"
+else
+	SRC_URI="https://github.com/LukeSmithxyz/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
```

Comment:
There's no need to drop existing arch support for simple version bumps. If there are major changes, ie API reworks or build system change, then you should drop the keywords to only what you can test. But normally just resetting the keywords to `~` will suffice on simple version bumps. Use **ekeyword ~all new-version.ebuild** to be sure!
https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades

Reply:
Fixed thank you!